framework/include/transfers/MultiAppTransfer.h

```diff
@@ -44,10 +44,24 @@ class MultiAppTransfer : public Transfer
   };
 
   /// Used to construct InputParameters
-  static MooseEnum directions() { return MooseEnum("to_multiapp from_multiapp"); }
+  static std::string possibleDirections() { return "to_multiapp from_multiapp"; }
```

Comment:
Not sure what to do with this. I need a way to initialize a `MooseEnum` (`_current_direction` and `_direction`) and a `MultiMooseEnum` (`_directions`)

Reply:
Usually we just hardcode these into the code... but this should be fine.

Hardcoding them sounds bad - but actually... it places the strings closer to where they could change.